client/src/views/Tests/EditTest.tsx

```diff
@@ -0,0 +1,52 @@
+import { models } from '@yahalom-tests/common';
+import React, { useState } from 'react'
+import { SectionNavigator, Section } from '../../components'
+import { TestDetails, TestEmails, TestQuestions } from "./TestForm";
+
+const EditTest: React.FC = () => {
+    const [test, setTest] = useState<models.dtos.TestDto>({
+        title: "",
+        intro: "",
+        questions: [],
+        language: models.enums.Language.Hebrew,
+        teacherEmail: "",
+        minPassGrade: 55,
+        isReviewEnabled: true,
+        successMessage: "",
+        failureMessage: "",
+        successEmail: { body: "", subject: "" },
+        failureEmail: { body: "", subject: "" },
+    });
+
+    const onChange = ({ questions }: Partial<Pick<models.dtos.TestDto, "questions">>) => {
+        ///questions are not undefined or empty
+        if (!questions || !(questions?.length > 0)) {
+            console.log("empty questions");
+        } else {
+            test.questions = questions!;
+            setTest({ ...test });
+            console.log(test.questions); //temporary - verify test contains the questions.
+        }
+    };
```

Comment:
Since this uses the `Partial`, you can't know there are any questions, so this is incorrect.
This function should be:
```suggestion
    const onChange = (changed: Partial<models.dtos.TestDto>) => {
        setTest({ ...test. ...changed });
        console.log(test.questions); //temporary - verify test contains the questions.
    };
```